index.html

```diff
@@ -1,34 +1,204 @@
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
+                        <button type="submit" class="submitBtn">See this project<img
+                                src="./images/Union.png" alt="direction image"
+                                id="arrow"></button>
+                    </div>
+                </div>
+
+                <img src="./images/Rectangle 21 (3).png" alt="image"
+                    class="imgCard4">
+
+                <div class="card4">
+                    <h2>Project name goes here</h2>
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
+                        <button type="submit" class="submitBtn">See this project<img
+                                src="./images/Union.png" alt="direction image"
+                                id="arrow"></button>
+                    </div>
+                </div>
+
+                <img src="./images/Rectangle 21 (5).png" alt="image"
+                    class="imgCard5">
+
+                <div class="card5">
+                    <h2>Project name goes here</h2>
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
+                        <button type="submit" class="submitBtn">See this project<img
+                                src="./images/Union.png" alt="direction image"
+                                id="arrow"></button>
+                    </div>
+                </div>
+
+                <img src="./images/Rectangle 21 (4).png" alt="image"
+                    class="imgCard6">
+
+                <div class="card6">
+                    <h2>Project name goes here</h2>
+                    <div class="buttons">
+                        <div class="projectBtn">
+                            <button type="button"  class="btn1">HTML/CSS</button>
+                            <button type="button"  class="btn2">Ruby on Rails</button>
+                            <button type="button"  class="btn3">JavaScript</button>
+                        </div>
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/198421117-59401367-4809-48d9-b984-2f69b91ee861.png)
Please wrap your list of technologies inside an `ul`, with each technology being an `li`. Also, kindly change your `buttons` to `divs`. This will make your project semantically accurate and improve your webpage accessibility 😉. Keep in mind that [buttons](https://www.w3schools.com/tags/tag_button.asp) in HTML should be linked to an action, for normal texts/cards like this, it is preferable to use divs.
You could have something like this:

```html
<ul class="technologies">
  <li> <div>HTML/CSS</div> </li>
  <li> <div>JavaScript</div> </li>
</ul>
```

Reply:
![Screenshot 2022-10-28 at 13 38 59](https://user-images.githubusercontent.com/37203742/198590355-e32068bd-5aed-43f9-9dac-174292c75e00.png)